### ISSUE TITLE
Do I have to build the dll?

### ISSUE THREAD

**Issue by funderburkjim** [dahlia/libsass-python#49]
I installed (pip install libsass) , and got 'Successfully installed'.  However, there is an error when
'import sass':
```
$ python -i
Python 2.7.9 (default, Dec 10 2014, 12:24:55) [MSC v.1500 32 bit (Intel)] on win
32
Type "help", "copyright", "credits" or "license" for more information.
>>> import sass
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "c:\Users\Jim\djangoproject\DJANGO\lib\site-packages\sass.py", line 22, i
n <module>
    from _sass import (OUTPUT_STYLES, compile_dirname,
ImportError: DLL load failed: The specified module could not be found.
>>> quit()
```

Am I supposed to compile the C++ code myself and install it somewhere?
(The OS is Windows 7).

**Comment by asottile** [comment]
I believe the "dll" it is referring to is _sass.pyd (built in this project).  Seeing the pip install output might be useful

**Comment by pakal** [comment]
Just had the same problem, on a python3.4 (32bits) on a win7-64bits OS.

Dependency Walker showed that "_sass.pyd" needed VCRUNTIME140.DLL and some other DLLs.

In  the end, just install the small "Visual C++ Redistributable for Visual Studio 2015" to get that bunch of missing C++ components. No need to install the 11+ GB of Visual Studio Suite ^^

**Comment by asottile** [comment]
This is fixed in 0.9.3, I'll upload the windows wheels once appveyor finishes

**Comment by asottile** [comment]
The wheels should be there now
